system/jhtree/jhtree.cpp

```diff
@@ -1527,11 +1527,15 @@ const CJHSearchNode *CKeyIndex::locateFirstNode(KeyStatsCollector &stats)
     return cur;
 }
 
-const CJHSearchNode *CKeyIndex::locateLastNode(KeyStatsCollector &stats)
+const CJHSearchNode *CKeyIndex::locateLastLeafNode(KeyStatsCollector &stats)
 {
     keySeeks++;
     stats.seeks++;
 
+    //Unusual - an index with no elements
```

Comment:
@jakesmith this is the only change, which makes it consistent with the firstLeaftNode() function above.
The other changes rename the functions to make it clearer what the function should do.